regression-tests/18-compile-arm-ports/Makefile

```diff
@@ -54,6 +54,8 @@ sensniff/ev-aducrf101mkxz \
 cfs-coffee/cc2538dk \
 cfs-coffee/openmote-cc2538 \
 cfs-coffee/zoul \
+fat/zoul:BOARD=remote-reva \
```

Comment:
I have not checked the schematic of the rev. A. Does it support SD, but not its power switch, or is SD not supported at all for this revision? In the former case, we can simply have a different code handling the power for rev. A. In the latter case, we should remove all the SD stuff for rev. A, not only in the READMEs and the tests, but also in the MAKEFILEs and in the code (but maybe not in `board.h`).